cypress/integration/entitybrowser_spec.js

```diff
@@ -5,28 +5,49 @@ describe('Entity Browser', function() {
     cy.login()
   })
 
-  beforeEach(() => {
-    cy.visit(`${helpers.getStoryUrl(['Apps', 'Entity Browser'], 'Entity Browser')}`)
-  })
-
   describe('ListView', () => {
+    beforeEach(() => {
+      cy.visit(`${helpers.getStoryUrl(['Apps', 'Entity Browser'], 'Entity Browser')}`)
+    })
+
     it('should load and display essential parts of the list view', function() {
+      cy.contains('Daten werden geladen...', {timeout: 15000})
       cy.get('[title="Vorname"]')
       cy.get('[type="submit"]')
+      cy.contains('Einträgen')
       cy.contains('Aktionen auf Alle')
-      cy.get('[data-list-cell]').should('have.length.above', 1)
+      cy.get('[data-cy=list-cell]', {timeout: 6000})
+        .should('have.length.above', 10)
+    })
+
+    describe('SearchForm', () => {
+      it('should display extended search form', function() {
+        cy.get('[data-cy=cypress-form-field]')
+          .should('have.length', 1)
+        cy.get('[data-cy=extend-search-button]')
+          .click()
+        cy.get('[data-cy=cypress-form-field]')
+          .should('have.length.above', 10)
+      })
+
+      it('should reset list view through search button reset', function() {
+        cy.get('input#input-searchForm-txtFulltext')
+          .type('Test Person')
+        cy.get('[data-cy=search-form_reset-button]')
+          .click({force: true})
+        cy.get('[data-cy=cypress-form-field]').children().children()
+          .should('be.empty')
+      })
     })
   })
 
   describe('DetailView', () => {
     beforeEach(() => {
-      cy.get('[data-list-cell]').first()
-        .click()
+      cy.visit(`${helpers.getStoryUrl(['Apps', 'Entity Browser'], 'Entity Browser')}detail/13477`)
```

Comment:
13477? Very random.
Good: Create a new user before every test run and test with this one. This will make sure this record really exists.
ok for the moment: Create a record by hand and make clear everyone sees that this is a record for testing and should not be deleted.